packages/core/src/Stores/notification-store.js

```diff
@@ -128,6 +130,16 @@ export default class NotificationStore extends BaseStore {
                 }
             }
         );
```

Comment:
This reaction looks a little weird, but it's better in my opinion to use reaction instead of adding `is_notifications_loading_completed = false` for every changes of notifications. Also, notifications uses in different stores and it's hard to maintain it along all the project code. Anyway, waiting for advice and any help from you